sdk/rust/Cargo.toml

```diff
@@ -3,6 +3,7 @@ name = "spin-sdk"
 version = { workspace = true }
 authors = { workspace = true }
 edition = { workspace = true }
+rust-version = "1.64"
```

Comment:
Curious to understand why we went with 1.64. I remember us fixing some lint errors last Spin release because rust 1.66 was released the same day. Could we use that version or is another reason we picked 1.64 here?

Reply:
1.64 is the MSRV for Spin itself, so this follows that.  And... if everything builds and works with 1.64, why force people to move to the bleeding edge?  This doesn't _stop_ people using 1.66 or 1.67 or whatever, as we do ourselves - it just provides a better error message if they are on a version of Rust that is unable to build the SDK.

Reply:
sgtm!